<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame_uart">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>221</width>
      <height>201</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_comid">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_box_com">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_baud">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_box_baud">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_data">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_box_data">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_stop">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_box_stop">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_parity">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_box_parity">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_uart_open">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开串口</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_uart_close">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭串口</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_info">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_led">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>2</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>LED控制</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_led_on">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>0</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开LED</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_led_off">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>0</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭LED</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_beep">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>BEEP控制</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_beep_on">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>打开蜂鸣器</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_beep_off">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>30</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>关闭蜂鸣器</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_edit_recv">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>461</width>
       <height>181</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_stauts">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>内部状态区</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_refresh">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>刷新状态</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_reboot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>复位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_reboot">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>91</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>复位</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_filepath_choose">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>310</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_file_path">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>文件路径</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_box_filepath">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>310</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_filepath_update">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>340</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>文件更新</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_dev">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>221</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_dev_id">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>设备ID</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_edit_dev_id">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_test">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>481</width>
      <height>371</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_cmd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>自定义指令</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_edit_cmd">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>381</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_send_cmd">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>指令发送</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_test">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>调试窗口</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="text_edit_test">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>461</width>
       <height>221</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>92</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>清空数据</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_socket">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>300</y>
      <width>221</width>
      <height>211</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_server_ipaddr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_edit_ipaddr">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>60</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.251</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_server_port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>服务器Port</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_edit_port">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>8000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_socket_open">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>连接网络</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_socket_close">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>180</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>断开网络</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_socket_type">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>协议</string>
     </property>
    </widget>
    <widget class="QComboBox" name="combo_box_socket_type">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_local_ipaddr">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>本地IP</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="line_edit_local_ipaddr">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.105</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>730</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
